.licensure.yml

```diff
@@ -1,14 +1,57 @@
-author: ZomboDB, LLC
-ident: MIT
-email: zombodb@gmail.com
+change_in_place: true
 excludes:
   - Cargo.toml
-  - ".*\\.h"
-  - "\\.cargo/config"
+  - Cargo.lock
+  - LICENSE
+  - pgx-utils/assets/ansi.tmTheme
+  - ".envrc"
+  - ".gitignore"
+  - "flake\\..*"
+  - "logo.*"
+  - ".github/**/*"
+  - ".cargo/config"
   - "cargo-pgx/src/templates/*"
-  - ".github/workflows/*"
   - ".*\\.control"
   - ".*\\.md"
+  - ".*\\.nix"
   - ".*\\.yml"
-  - ".*\\.sh"
-  - load-order.txt
+licenses:
+  - files: any
+    ident: MIT
+    authors:
+      - name: ZomboDB, LLC and Technology Concepts & Design, Inc.
```

Comment:
According to the header below showing "Portions Copyright 2019-2021 ZomboDB, LLC." then later "Portions Copyright 2021-[year] Technology Concepts & Design, Inc. <support@tcdi.com>.", it seems incorrect to list `ZomboDB, LLC` as a current copyright holder? Should `ZomboDB, LLC` have their copyright header portion changed to include `2022`?

Reply:
This is the only one that needs to be fixed, and I'll do that right now.  This `licensure` tool is quite terrible and I hadn't had any caffeine this morning when I pushed this.